packages/@aws-cdk/cdk/lib/aspects/README.md

```diff
@@ -0,0 +1,292 @@
+## Aspects
```

Comment:
Thanks for writing the README, but I feel this doc is focusing on implementation perspective a little too much over customer perspective. I'm going to add some more comments over the course of it.